client/src/about/about.yaml

```diff
@@ -121,59 +119,77 @@ our_data_sub_title_2:
 
 
-    L'équipe d'InfoBase du GC est obsédée par la qualité des données et, à notre avis, cela commence avant le début de la collecte. À cette fin, nous avons développé une technologie qui permet de saisir
-    les données essentielles des ministères et avons mis en place des mécanismes pour assurer la qualité, l'exactitude et la fiabilité de toutes nos données. 
+    L'équipe chargée de l'analyse des dépenses est obsédée par la qualité des données. Nous avons la conviction que cela commence bien avant le début de la collecte de données. À cette fin, nous avons développé une technologie qui permet de saisir
+    les données essentielles des ministères et avons mis en place des mécanismes pour assurer la qualité, l'exactitude et la fiabilité des données. 
 
 
-    Nous savons qu'un chiffre ne dit pas toujours tout, c'est pourquoi InfoBase du GC vient en aide aux utilisateurs en leur présentant en complément des renseignements contextuels. 
-    Nous collaborons avec les organisations gouvernementales pour expliquer des données intéressantes telles qu'une hausse ou une baisse des dépenses, ou encore la situation concernant un résultat précis.
+    Nous savons qu'un chiffre ne dit pas toujours tout, c'est pourquoi l'InfoBase du GC vient en aide aux utilisateurs en leur donnant un complément d'information contextuelle. 
+    Nous collaborons avec les organisations gouvernementales pour expliquer des données intéressantes telles qu'une hausse ou une baisse des dépenses, ou encore l'état de la situation concernant un résultat précis.
 
-    Alors que la majorité des données d'InfoBase du GC sont accessibles aux utilisateurs sur le Portail de {{ext_link "données ouvertes" "https://ouvert.canada.ca/fr/donnees-ouvertes"}} du gouvernement du Canada, InfoBase du GC est la seule tentative d'intégration de ces ensembles de données dans un référentiel unique destiné aux parlementaires, aux fonctionnaires et à la population canadienne. À cette fin, InfoBase du GC est évolue constamment avec l'ajout de nouvelles sources de données, de visualisations et d’explications. <a href="#contact">Communiquez avec nous</a> si vous avez des données, des réflexions ou des idées à partager!
+    Alors que les ensembles de données sont accessibles aux utilisateurs sur le {{ext_link "Portail de données ouvertes" "https://ouvert.canada.ca/fr/donnees-ouvertes"}} et les rapports officiels du gouvernement du Canada, l'InfoBase du GC est la seule tentative d'intégration de ces ensembles de données dans un référentiel unique destiné aux parlementaires, aux fonctionnaires et à la population canadienne. À cette fin, l'InfoBase du GC évolue constamment avec l'ajout de nouvelles sources de données, de visualisations et d’explications. <a href="#contact">Communiquez avec nous</a> si vous avez des données, des réflexions ou des idées à partager!
```

Comment:
In the english section we changed it from "Canadian population" to "Canadians". Would we not want to do the same thing for the french section?

Reply:
Yeah please do!